resources/weapons/a2a-missiles/K-13A.yaml

```diff
@@ -0,0 +1,5 @@
+name: K-13A
+year: 1962
+fallback: 

```

Comment:
If there is no fallback the field should be omitted.

Reply:
Done